src/install/aim/agent-lang/python-config.mdx

```diff
@@ -0,0 +1,129 @@
+---
+componentType: default
+headingText: Configure the Python agent
+---
+
+You installed the Python agent in the last step, but you still need to apply certain configurations so the agent can collect AI-specific data. 
+
+1. We recommend applying these configurations from `newrelic.ini` config file. If you have a more complex set up with multiple environments, you can configure with environment variables.
+
+    <CollapserGroup>
+        <Collapser
+            id="config-file"
+            title="Enable with config file"
+        >
+
+    ```ini
+    ai_monitoring.enabled = true
+    ```   
+        </Collapser>
+        <Collapser
+            id="config-enviro"
+            title="Enable with environment variables"
+        >
+
+    ```bash
+    NEW_RELIC_AI_MONITORING_ENABLED=true
+    ```
+
+        </Collapser>
+    </CollapserGroup>
+
+2. These configurations adjust how the agent captures and sends AI data. They let you limit the number of events or spans the agent collects, increase or decrease message limits, or enable logging.  
+
+    <CollapserGroup>
+        <Collapser
+            id="py-event-limit"
+            title="ELEPHANT"
+        >
+
+        One instance of the Python agent sends a maximum of 100,000 events per minute. Once the agent hits that maximum, it starts to sample event data, giving a general representation of events across the harvest cycle. 
+
+        Set limit with config file: 
+
+            ```ini
+                event_harvest_config.harvest_limits.custom_event_data
+            ```
+
+        Set limit with environment variables: 
+
+            ```bash
+                NEW_RELIC_ML_INSIGHTS_EVENTS_MAX_SAMPLES_STORED
+            ```
+
+        * Setting this value to the maximum value (100k) means that no sampling occurs. However, ingest and memory storage will go up.
+        * Setting this to a lower value (1k or less) prompts the agent to sample, depending on the volume of your app. If you're concerned with cost or need less data in New Relic, setting a lower value is a good option. 
+
+        </Collapser>
+        <Collapser
+            id="py-event-harvest"
+            title="ELEPHANT"
+        >
+
+        Set a limit for how many sample spans the agent records. Spans capture when an operation occurred in your app code, how long it took to complete, and other operations it relates to. 
+
+        Set limit with config file:
+
+        ```ini
+            event_harvest_config.harvest_limits.span_event_data
+        ```              
+
+        Set limit with environment variables:
+
+        ```bash
```

Comment:
I'd like to continue the config vs enviro variable pattern for all configurations. could you provide the enviro variable pair for each config?

@hmstepanek